File: C#BasicsMarch2015/1.FirstStepInCoding/RectangleArea/RectangleArea/Rectangle.cs
using System;

namespace RectangleArea
{
    class Rectangle
    {
        static void Main(string[] args)
        {
            var a = decimal.Parse(Console.ReadLine());
            var b = decimal.Parse(Console.ReadLine());
            var width = Math.Abs(a);
            var height = Math.Abs(b);
            var area = width * height;
            Console.WriteLine(area);
        }
    }
}

File: JSAdvancedJune2017/13.Classes-and-Class-Members-Lab/rectangle.js
class Rectangle {
    constructor(width, height, color){
        this.width = width;
        this.height = height;
        this.color = color;
    }

    calcArea() {
        return this.height * this.width;
    };
}

let r = new Rectangle(5, 3, 'red');
console.log(r.calcArea());